lib/logstash/filters/fingerprint.rb

```diff
@@ -154,7 +165,8 @@ def anonymize_openssl(data)
       hash  = OpenSSL::HMAC.digest(@digest, @key, data.to_s)
       Base64.strict_encode64(hash).force_encoding(Encoding::UTF_8)
     else
-      OpenSSL::HMAC.hexdigest(@digest, @key, data.to_s).force_encoding(Encoding::UTF_8)
+      hash = OpenSSL::HMAC.hexdigest(@digest, @key, data.to_s).force_encoding(Encoding::UTF_8)
```

Comment:
Yes sorry, I only had that during my debugging.  Meant to take that out, will do that right now.